src/main/java/org/radarcns/management/web/rest/RoleResource.java

```diff
@@ -135,4 +139,13 @@
             .findOneByProjectNameAndAuthorityName(projectName, authorityName));
     }
 
+    /**
+     * Create a user-friendly display name for a given role. Useful for passing to the
+     * notification system
+     * @param role The role to create a user-friendly display for
+     * @return the display name
+     */
+    private String displayName(RoleDTO role) {
+        return role.getProjectName() + " - " + role.getAuthorityName();
```

Comment:
Can have `:` instead of `-`. So we ca reuse this method when necessary since it matches existing logics.

Reply:
Good point, will change.